tracetesting/run.bash

```diff
@@ -25,16 +25,14 @@ echo "Preparing to run tests on API..."
 echo ""
 
 echo "Environment variables considered on this run:"
-echo "TRACETEST_CLI_MAIN:      $TRACETEST_CLI_MAIN"
-echo "TARGET_URL:              $TARGET_URL"
-echo "TRACETEST_MAIN_ENDPOINT: $TRACETEST_MAIN_ENDPOINT"
-echo "DEMO_APP_URL:            $DEMO_APP_URL"
-echo "DEMO_APP_GRPC_URL:       $DEMO_APP_GRPC_URL"
-
-cat << EOF > .main.env
-TRACETEST_CLI_MAIN=$TRACETEST_CLI_MAIN
+echo "TRACETEST_CLI:      $TRACETEST_CLI"
+echo "TARGET_URL:         $TARGET_URL"
+echo "TRACETEST_ENDPOINT: $TRACETEST_ENDPOINT"
+echo "DEMO_APP_URL:       $DEMO_APP_URL"
+echo "DEMO_APP_GRPC_URL:  $DEMO_APP_GRPC_URL"
+
+cat << EOF > .env
 TARGET_URL=$TARGET_URL
-TRACETEST_MAIN_ENDPOINT=$TRACETEST_MAIN_ENDPOINT
```

Comment:
Since this variable is only used on this script, does not make sense to add it on `.env` file.